packages/modular-scripts/src/cli.ts

```diff
@@ -226,6 +226,22 @@ program
     await port(relativePath);
   });
 
+interface TypecheckOptions {
+  silent: boolean;
+}
+
+program
+  .command('typecheck')
+  .option(
```

Comment:
What's the use case for this? I would avoid adding any unnecessary APIs to this until we've got a better idea of when we'll need them.

Reply:
I added this as a way to silence error messages and provide a green/red light on typechecking. I'm impartial so I will remove for simplicity's sake.